controllers/complianceeventsapi/server.go

```diff
@@ -92,6 +181,357 @@ func (s *complianceAPIServer) Stop() error {
 	return nil
 }
 
+// splitQueryValue will parse a string and split on unescaped commas. Empty values are discarded.
+func splitQueryValue(value string) []string {
+	values := []string{}
+
+	var currentVal string
+	var previousChar rune
+
+	for _, char := range value {
+		if char == ',' && previousChar != '\\' {
```

Comment:
Do you have a unit test for this? I am confused that previousChar could be '\\'? Isn't '\\' 2 chars?

Reply:
'\\' is actual one character because the first backslash is an escape character. I realized this function wasn't removing the escape character so in my push I'll be adding a unit test to catch this.

Reply:
Unit test added?

Reply:
Sorry, I forgot to `git add` the new test file.